crates/store/src/state.rs

```diff
@@ -1088,9 +1113,22 @@ async fn load_mmr(db: &mut Db) -> Result<Mmr, StateInitializationError> {
 }
 
 #[instrument(level = "info", target = COMPONENT, skip_all)]
-async fn load_account_tree(db: &mut Db) -> Result<AccountTree, StateInitializationError> {
+async fn load_account_tree(
+    db: &mut Db,
+    _block_num: BlockNumber,
+) -> Result<AccountTree<LargeSmt<MemoryStorage>>, StateInitializationError> {
```

Comment:
Should we create an alias for this (or maybe even a newtype)? Maybe `InMemoryAccountTree`? In the future, we'd maybe have something like `PersistentAccountTree`.